3-extra/card-validator.js

```diff
@@ -0,0 +1,32 @@
+//creating card validator function
+
+function creditCardNumberValidator(cardNumber) {
+  let cardNumberElement = cardNumber.toString().split("");
+  if (!cardNumberElement.every((element) => /[0-9]/.test(element))) {
+    return false + "Your card is invalid!";
+  }
+  if (cardNumberElement.length !== 16) {
+    return false + ", Your card is invalid!";
+    //checks if card number has 16 digits.
+  }
+  if (cardNumberElement.every((element) => element !== element[0])) {
+    return false + ", Your card is invalid!";
+  }
+  if (cardNumberElement[cardNumberElement.length - 1] % 2) {
+    return false + ", Your card is invalid!";
+  }
+  //checks if the last digit of the card number i an even number.
```

Comment:
Generally the convention (at least I've always seen it) is to put the comment above the line of code that it's explaining, rather than below.